community/kernel/src/main/java/org/neo4j/kernel/impl/store/NeoStore.java

```diff
@@ -152,6 +157,19 @@ public static boolean isStorePresent( FileSystemAbstraction fs, File storeDir )
     private final int relGrabSize;
     private final CappedOperation<Void> transactionCloseWaitLogger;
 
+    private final ThreadLocal<AtomicReference<StoreStatement>> storeStatement = new ThreadLocal<AtomicReference<StoreStatement>>()
```

Comment:
Perhaps is it better to move the `StoreStatement` to `KernelTransactionImplementation`, since those are already pooled?